test/libman.Test/LibmanInstallTest.cs

```diff
@@ -186,8 +186,7 @@ public void TestInstall_WithExistingLibmanJson_SpecificFiles()
 }";
 
             File.WriteAllText(Path.Combine(WorkingDir, "libman.json"), initialContent);
-
-            int result = command.Execute("jquery@3.2.1", "--files", "jquery.min.js");
+            _ = command.Execute("jquery@3.2.1", "--files", "jquery.min.js");
```

Comment:
>_ =  [](start = 12, length = 4)

Nit: Do we need this discard at all? It's not async, right? Also, last method doesn't use a discard, so we should be at least consistent.

Reply:
I don't think we do.  There was a squiggle and the default lightbulb was to replace the unused local with a discard, but you're right we can just remove the assignment.  I'll clean this up.